Yosemite/Yosemite/Model/WooPlans/WooPlan.swift

```diff
@@ -276,15 +330,35 @@ private extension WooPlan {
     }
 
     enum Localization {
-        static let essentialPlanName = NSLocalizedString(
-            "Woo Express Essential Monthly",
-            comment: "Title for the plan on a screen showing the Woo Express Essential Monthly plan upgrade benefits")
+        static let essentialPlanNameFormat = NSLocalizedString(
+            "Woo Express Essential %1$@",
```

Comment:
🤔 In this string and similar others, should these be `NSLocalizedStrings`? Or we want the name to be `Woo Express Essential monthly/yearly` in all languages? Or perhaps translate only the monthly/yearly part?